backend/src/db/migrations/20251021112356_add-certificate-auto-renewal.ts

```diff
@@ -0,0 +1,51 @@
+import { Knex } from "knex";
```

Comment:
IGNORE PLACEMENT OF COMMENT

I think this should just be "Renewal" here instead of "Auto-Renewal", primarily because we have renamed/repurposed the column

<img width="1500" height="547" alt="image" src="https://github.com/user-attachments/assets/e213b704-c78e-417b-8771-5eb2cf8c6473" />